chunky/src/java/se/llbit/chunky/world/CubicWorld.java

```diff
@@ -51,7 +51,10 @@ protected synchronized File getRegionDirectory(int dimension) {
   }
 
   @Override
-  public ChunkData createChunkData() {
+  public ChunkData createChunkData(ChunkData chunkData, int chunkVersion) {
```

Comment:
Is the interface's javadoc already updated to make clear that it _may or may not_ re-use it and ensures that it returns a compatible `ChunkData` implementation instance?